test/performance/broker-latency/broker-latency.yaml

```diff
@@ -32,8 +32,9 @@ spec:
       restartPolicy: Never
       containers:
       - name: sender-receiver
-        image: knative.dev/eventing/test/test_images/latencymako_sender_receiver
+        image: knative.dev/eventing/test/test_images/latencymako
         args:
+        - "--role=sender-receiver"
```

Comment:
This flag determines whether the image runs the sender-receiver or the aggregator.